vercel/README.md

```diff
@@ -0,0 +1,52 @@
+# Agent Check: Vercel
+
+## Overview
+
+[Vercel][1] is a deployment and collaboration platform that enables frontend developers to build high-performance websites and applications. Vercel is also the creator of Next.js, a React framework developed in collaboration with engineers at Google and Facebook in 2016. Vercel users can leverage a built-in deployment tool that manages the build and rendering process, as well as a proprietary [Edge Network][2] that caches their sites for fast retrieval. Additionally, Vercel offers [Serverless Functions][3], which allow users to deploy serverless code to accomplish essential backend processes like user authentication, form submission, and database queries.
+
+Integrate Vercel with Datadog to:
+
+* View and parse your application logs using [Datadog’s Log Management][4]
+* See the number of requests and 4xx/5xx HTTP errors to your serverless applications and APIs running on Vercel
+* Monitor frontend performance with [Datadog Synthetics][5]
+
+## Setup
+
+To set up the Vercel-Datadog integration:
+
+* Generate a [Datadog API key][6]
+* Configure the logs integration through the [Vercel Marketplace][7]
+
+
+### Installation
+
+To install the Vercel check on your host:
```

Comment:
are there supposed to be instructions following?

Reply:
The instructions are above - this was a copy paste error